### PR TITLE
Slovak language

### PR DESCRIPTION
**Pull request by BIIINO**
Hello, 

there is almost 100% translated Slovak language - except few words from installation wizard which are beter understand in english.

Also creating pull request for mobile app right now.

Thank you in advance

### CODE REVIEW

**Comment by BIIINO** [comment]
Mobile app version:

https://github.com/bytefury/crater-mobile/pull/14

File: resources/assets/js/views/reports/ExpensesReport.vue [hunk 1]
```diff
@@ -92,19 +92,19 @@ export default {
...
-        'This Week',
-        'This Month',
-        'This Quarter',
-        'This Year',
-        'Previous Week',
-        'Previous Month',
-        'Previous Quarter',
-        'Previous Year',
-        'Custom',
+        'Dnes',
```
Comment (mohitpanjwani) [inline resources/assets/js/views/reports/ExpensesReport.vue:95]: @BIIINO Hey, changing this list directly will update it for all languages (it'll show date range fields in Slovak even if you select english language)

In order to make this date range fields translatable, we need to add all of these strings as keys on `en.json` etc.

File: resources/assets/js/views/reports/ProfitLossReport.vue [hunk 2]
```diff
@@ -92,19 +92,19 @@ export default {
...
-        'Previous Year',
-        'Custom',
+        'Dnes',
+        'Tento týždeň',
+        'Tento mesiac',
+        'Tento štvrťrok',
+        'Tento rok',
+        'Minulý týždeň',
+        'Minulý mesiac',
+        'Minulý štvrťrok',
```
Comment (mohitpanjwani) [inline resources/assets/js/views/reports/ProfitLossReport.vue:102]: Same issue here.

File: resources/assets/js/views/reports/SalesReports.vue [hunk 3]
```diff
@@ -104,22 +104,22 @@ export default {
...
-        'This Quarter',
-        'This Year',
-        'Previous Week',
-        'Previous Month',
-        'Previous Quarter',
-        'Previous Year',
-        'Custom',
+        'Dnes',
+        'Tento týždeň',
+        'Tento mesiac',
```
Comment (mohitpanjwani) [inline resources/assets/js/views/reports/SalesReports.vue:112]: Same issue here.

File: resources/assets/js/views/reports/TaxReport.vue [hunk 4]
```diff
@@ -85,18 +85,18 @@ export default {
...
-        'Previous Quarter',
-        'Previous Year',
-        'Custom',
+        'Dnes',
+        'Tento týždeň',
+        'Tento mesiac',
+        'Tento štvrťrok',
+        'Tento rok',
+        'Minulý týždeň',
+        'Minulý mesiac',
```
Comment (mohitpanjwani) [inline resources/assets/js/views/reports/TaxReport.vue:94]: Same issue here.

**Review by mohitpanjwani — CHANGES_REQUESTED** [review]
Hey, Thank you for contributing. 

Please remove the changes from Report pages or add them as translatable keys so we can review this further :)

**Comment by mohitpanjwani** [comment]
Also I've just merged a PR for Vietnamese, so can you also fix those minor conflicts 👍

**Comment by BIIINO** [comment]
Hi,

how can i add all these strings as translatable keys? For example:
-----------------------------------
"this_year": "Tento rok",     
-----------------------------------
Is that okay format ?

Thank you

**Comment by mohitpanjwani** [comment]
> Hi,
> 
> how can i add all these strings as translatable keys? For example:
> 
> "this_year": "Tento rok",
> 
> Is that okay format ?
> 
> Thank you

Hey, yes that would be great. 

For now you can just add the key values for english and Slovak. 

Other languages which don't have this key defined, should automatically fallback to english.

**Comment by BIIINO** [comment]
Tested on local installation but not recognized, removing these files.

**Comment by BIIINO** [comment]
i am still trying to add for example these as transable key into en.json.

When i create line in en.json:
"this_year": "This year",
and
"this_year": "Tento rok",


nothing changed on web, still english. What else is necesery to be done for correct implementation ?

**Comment by BIIINO** [comment]
@mohitpanjwani hint me please :)

**Pull request merged**